go-concourse/concourse/pipelines.go

```diff
@@ -57,6 +57,28 @@ func (team *team) OrderingPipelines(pipelineNames []string) error {
 	}, &internal.Response{})
 }
 
+func (team *team) OrderingPipelinesWithinGroup(groupName string, instanceVars []atc.InstanceVars) error {
```

Comment:
We switched go-concourse to use the now preferred `HttpAgent` rather than the `Connection`, and special case `http.StatusBadRequest` so that we can display a more friendly error message